src/Maestro/Maestro.Data/Models/Build.cs

```diff
@@ -147,7 +147,14 @@ public string GitHubBranch
         /// If true, the build has been released to the public. This can be used to make decisions on whether certain
         /// builds should be included in future release drops.
         /// </summary>
-        public bool Released { get; set; } = false;
+        public bool Released { get; set; }
+
+        /// <summary>
+        /// If true, the build was marked with `$(DotNetFinalVersionKind) == 'release'`. Which means that it 
+        /// produced assets with only (Major).(Minor).(Patch) version.
+        /// More info is available here: https://github.com/dotnet/arcade/blob/master/Documentation/CorePackages/Versioning.md#build-kind
+        /// </summary>
+        public bool Stable { get; set; }
```

Comment:
I can't remember...what was the reason to include this? Feeding it to publishing? Isn't that already in the manifest?

Reply:
We use this as a property in a task defined here [SetupTargetFeeds.proj](https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/SdkTasks/SetupTargetFeeds.proj) to decide what kind of feed we are going to publish to. That task is called from the YAML channel stages.

I'm adding this to BAR so that we can recover it later when promoting a build and flow the property through the YAML/PROJ files the same way as we do at build creation time.

This is the issue tracking the work btw: https://github.com/dotnet/arcade/issues/3834